app/react/docker/services/ListView/ServicesDatatable/columns/ports.tsx

```diff
@@ -36,8 +36,9 @@ function Cell({
 
   return ports
     .filter((port) => port.PublishedPort)
-    .map((port) => (
+    .map((port, index) => (
       <PublishedPortLink
+        key={index}
```

Comment:
go for `key={port.PublishedPort}` instead of `key={index}`.

In this case, there won't be much behaviour difference, but for arrays that can change sorting etc, React can behave in weird ways when the index is used as a key